ivadomed/scripts/automate_training.py

```diff
@@ -34,11 +34,15 @@
 
 def get_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-c", "--config", required=True, help="Base config file path.")
-    parser.add_argument("-p", "--params", required=True, help="JSON file where hyperparameters to experiment are "
-                                                              "listed.")
+    parser.add_argument("-c", "--config", required=True, help="Base config file path.",
+                        metavar=Metavar.file)
+    parser.add_argument("-p", "--params", required=True,
+                        help="""JSON file where hyperparameters to experiment are
```

Comment:
Just out of curiosity: why use the triple vs single quotes (""" vs "). I see some help you modified have single quotes  ( line 44) and some triple quotes (line 40).

Reply:
Oh it's just cause the triple quotes allow for multi-line quotes, so you don't have to use new quotes with each line :)